src/System.Runtime.Analyzers/UnitTests/CallGCSuppressFinalizeCorrectlyTests.cs

```diff
@@ -16,5 +30,568 @@ protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
             return new CSharpCallGCSuppressFinalizeCorrectlyAnalyzer();
```

Comment:
Did you add any code to the C# and VB specific derived types for this analyzer? Otherwise, please delete them, change CallGCSuppressFinalizeCorrectlyAnalyzer to be a sealed type with DiagnsoticAnalyzerAttribute(C#, VB).